src/debugger/cli-debugger.c

```diff
@@ -114,6 +118,10 @@ static struct CLIDebuggerCommandSummary _debuggerCommands[] = {
 	{ "watch/c", _setWriteChangedWatchpoint, "Is", "Set a change watchpoint" },
 	{ "watch/r", _setReadWatchpoint, "Is", "Set a read watchpoint" },
 	{ "watch/w", _setWriteWatchpoint, "Is", "Set a write watchpoint" },
+	{ "watchr", _setReadWriteRangeWatchpoint, "IIs", "Set a range watchpoint" },
+	{ "watchr/c", _setWriteChangedRangeWatchpoint, "IIs", "Set a change range watchpoint" },
+	{ "watchr/r", _setReadRangeWatchpoint, "IIs", "Set a read range watchpoint" },
+	{ "watchr/w", _setWriteRangeWatchpoint, "IIs", "Set a write range watchpoint" },
```

Comment:
`watchr` is not self-documenting. Perhaps the more verbose name `watch-range` and add `watchr` as aliases in the alias table below.